src/Microsoft.TestPlatform.ObjectModel/Microsoft.TestPlatform.ObjectModel.csproj

```diff
@@ -6,6 +6,7 @@
     <TargetFrameworks>net7.0;$(NetFrameworkMinimum);$(NetCoreAppMinimum);netstandard2.0;</TargetFrameworks>
     <!-- Suppress warning that is caused by Nuget.Frameworks code that we copy. -->
     <NoWarn>$(NoWarn);SYSLIB0051</NoWarn>
+    <PublishTrimmed Condition="'$(TargetFramework)'=='net7.0' OR '$(TargetFramework)'=='$(NetCoreAppMinimum)'">true</PublishTrimmed>
```

Comment:
Note to self: Check if IsAotCompatible or EnableTrimAnalyzer is a better option here